README.md

```diff
@@ -18,6 +18,25 @@ in AWS.
 
 ## Installing cloudlift
 
+### Using Script
+
+```sh
+curl -LO https://raw.githubusercontent.com/GetSimpl/cloudlift/master/scripts/installer.sh    
+chmod +x installer.sh
+sudo ./installer.sh
+```
+
+Note: This will download the `installer.sh` script to the current directory and run it. Run `installer.sh --help` for more options.
```

Comment:
```
sh -c "$(curl -fsSL https://raw.githubusercontent.com/GetSimpl/cloudlift/master/scripts/installer.sh)"
```

this will do, just need to append `sudo` wherever required. and yes, the installer is compatible with `sh`.

Reply:
We don't need to prepend with`sudo` anymore, since the installer now supports implicit sudo